exercises/J-functions/exercise.js

```diff
@@ -1,7 +1,7 @@
 function halve(number) {
-  // complete the function here
+  return number - 6;
```

Comment:
This won't halve the number. It will subtract it by 6. You'd need instead this:

return number / 2;